Observer/CreditmemoRefundObserver.php

```diff
@@ -114,20 +113,20 @@ public function execute(\Magento\Framework\Event\Observer $observer)
             //If payment method belongs to terminal codes
             if (in_array($payment->getMethod(), SystemConfig::getTerminalCodes())) {
                 //Create orderlines from order items
-                $orderLines = $this->processRefundOrderItems($storePriceIncTax, $memo);
+                $orderLines = $this->processRefundOrderItems($memo);
                 //Send request for payment refund
                 $this->sendRefundRequest($memo, $orderLines, $orderObject, $payment, $storeCode);
             }
         }
     }
 
     /**
-     * @param $memo
+     * @param object $memo
```

Comment:
is this not \Magento\Sales\Api\Data\CreditmemoInterface?

Reply:
Yes, missed it. Done